src/q8/devmeth-INGI2251/exam/2016/Juin/All/parts/q01.tex

```diff
@@ -83,33 +101,34 @@ \subsection{The spiral model}
 
 \begin{figure}[!ht]
     \centering
-    \includegraphics[width=0.8\linewidth]{spiral_model.png}
+    \includegraphics[width=0.5\linewidth]{spiral_model.png}
     \caption{Spiral model lifecycle}
 \end{figure}
 \FloatBarrier{}
 
-\subsection{Agile model}
-The classical software development process models are rigorous process for
-software conception, documentation, development and testing. The alternative is the agile method that is a flexible process that can adapt to changing requirements. The overall goal of agile development is to satisfy the customer by \enquote{early and continuous delivery of valuable software}.
-
-\paragraph{Agile Manifesto :\\}
+\subsubsection{Agile model}
 
 \begin{itemize}
-    \item Value individuals and interactions over processes and tools
-    \item Prefer to invest time in producing working software rather than in producing comprehensive documentation
-    \item Focus on customer collaboration rather than contract negotiation
-    \item Concentrate on responding to change rather than on creating a plan and then following it
+    \proitem{} Flexible process (opposite to rigorous process with
+conception/documentation/dev/tests) that adapt to changing requirements.
+    \item The overall goal of agile development is
+to satisfy the customer by \enquote{early and continuous delivery of
+valuable software}.
 \end{itemize}
 
+(Extreme programming (XP), Crystal, Scrum)
+
 
-\paragraph{Examples of agile processes :\\}
+\paragraph{Agile Manifesto:}
 
 \begin{itemize}
-    \item Extreme programming (XP)
-    \item Crystal
-    \item Scrum
+    \item Value \textbf{individuals} and \textbf{interactions} over processes and tools
+    \item Prefer producing \textbf{working software} rather than comprehensive documentation
+    \item Focus on \textbf{customer collaboration} rather than contract negotiation
+    \item Concentrate on \textbf{responding to change} rather than on creating a plan and then following it
 \end{itemize}
 
```

Comment:
Ici il faudrait rajouter le fait que c'est du développement itératif et incrémental (il attend ces deux mots si tu tombes sur cette question)